src/components/TextEditor/extensions/markdown-paste-extension.ts

```diff
@@ -2,6 +2,8 @@ import { Extension } from '@tiptap/core'
 import { Plugin, PluginKey } from '@tiptap/pm/state'
```

Comment:
this extension was originally supposed to handle markdown paste only. now that you've added image processing you should rename the extension name.